extensions/common/vault/vault-hashicorp/src/test/java/org/eclipse/edc/vault/hashicorp/client/HashicorpVaultClientIntegrationTest.java

```diff
@@ -37,22 +39,31 @@
 
 @ComponentTest
 @Testcontainers
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
not needed

Reply:
thanks for pointing this out ... I took over your gratious blueprint.